seed/lib/xml_mapping/mapper.py

```diff
@@ -8,115 +8,15 @@
 from __future__ import absolute_import
```

Comment:
Since this has gotten so slim and imports heavily from building_sync.mapping, what's your thoughts on bringing this into the building_sync/mappings.py file. 

Per our conversation - we can probably do this later, but leaving this here as a note/documentation.

Reply:
I think I'll hold off on this, but do agree that it would be nice to consolidate and reorganize a few of these files